<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QComboBox</class>
 <widget class="QWidget" name="QComboBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QComboBox</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>简单的ComboBox</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="initiallist">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>初始化列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearlist">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>清除列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Enabled">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>可编辑</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox1">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>上海市</string>
           </property>
           <property name="icon">
            <iconset resource="QComboBox.qrc">
             <normaloff>:/images/images/aim.ico</normaloff>:/images/images/aim.ico</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>北京市</string>
           </property>
           <property name="icon">
            <iconset resource="QComboBox.qrc">
             <normaloff>:/images/images/aim.ico</normaloff>:/images/images/aim.ico</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>厦门市</string>
           </property>
           <property name="icon">
            <iconset resource="QComboBox.qrc">
             <normaloff>:/images/images/aim.ico</normaloff>:/images/images/aim.ico</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>有用户数据的ComboBox</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="initial_city_num">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>初始化城市和区号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>15</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="QComboBox.qrc"/>
 </resources>
 <connections/>
</ui>
